packages/npm-package-json-lint-config/test/index.test.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * External dependencies
+ */
+import isPlainObj from 'is-plain-obj';
+
+/**
+ * Internal dependencies
+ */
+import config from '../';
+
+describe( 'npm-package-json-lint config tests', () => {
+	it( 'should be an object', () => {
+		expect( isPlainObj( config ) ).toBeTruthy();
```

Comment:
The `toBeTruthy` assertion should be discouraged, as it implies that we expect it could be a value other than explicitly `true` (a non-zero number, a non-empty string, etc). If we expect it should be explicitly `true`, we should assert as `toBe( true )`.